### PR TITLE
Fix tests on django 1 9 continued

### PR DESCRIPTION
**Pull request by yoshrote**

### CODE REVIEW

**Comment by yoshrote** [comment]
there are still some broken tests. a number of which are around the order of parameters in the query string.....

File: tastypie/serializers.py [hunk 1]
```diff
@@ -251,6 +251,7 @@ def to_etree(self, data, options=None, name=None, depth=0):
                 element = Element('objects')
             for item in data:
                 element.append(self.to_etree(item, options, depth=depth+1))
+                element[:] = sorted(element, key=lambda x: x.tag)
```
[the remark this answers is not in the document]
Comment (yoshrote) [reply tastypie/serializers.py]: [nope](https://github.com/django-tastypie/django-tastypie/blob/master/tastypie/serializers.py#L342)

File: tastypie/serializers.py [hunk 2]
```diff
@@ -259,11 +260,13 @@ def to_etree(self, data, options=None, name=None, depth=0):
                 element.set('type', 'hash')
             for (key, value) in data.iteritems():
                 element.append(self.to_etree(value, options, name=key, depth=depth+1))
+                element[:] = sorted(element, key=lambda x: x.tag)
```
[the remark this answers is not in the document]
Comment (yoshrote) [reply tastypie/serializers.py]: [still nope](https://github.com/django-tastypie/django-tastypie/blob/master/tastypie/serializers.py#L352)

File: tastypie/serializers.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (slinkp) [reply tastypie/serializers.py]: what the ...

that's just bizarre

File: tastypie/serializers.py [hunk 3]
```diff
@@ -259,11 +260,13 @@ def to_etree(self, data, options=None, name=None, depth=0):
...
         elif isinstance(data, Bundle):
             element = Element(name or 'object')
             element.set('type', 'hash')
             for field_name, field_object in data.data.items():
                 element.append(self.to_etree(field_object, options, name=field_name, depth=depth+1))
+                element[:] = sorted(element, key=lambda x: x.tag)
```
[the remark this answers is not in the document]
Comment (yoshrote) [reply tastypie/serializers.py]: [nope again](https://github.com/django-tastypie/django-tastypie/blob/master/tastypie/serializers.py#L358)

File: tests/validation/tests.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 from django.conf import settings
 from django.core.urlresolvers import reverse, NoReverseMatch
 from django.http import HttpRequest
-from django.test import TestCase
+from testcases import TestCaseWithFixture as TestCase
```
[the remark this answers is not in the document]
Comment (yoshrote) [reply tests/validation/tests.py]: turning this into a relative import seems to completely break `tox`

**Pull request merged**